resource_customizations/argoproj.io/AnalysisRun/health.lua

```diff
@@ -0,0 +1,32 @@
+hs = {}
+if obj.status ~= nil then
+    if obj.status.status == "Pending" then
+        hs.status = "Progressing"
+        hs.message = "Waiting for analysis run to finish: status is pending."
+    end
+    if obj.status.status == "Running" then
+        hs.status = "Progressing"
+        hs.message = "Waiting for analysis run to finish: status is running."
+    end
+    if obj.status.status == "Successful" then
+        hs.status = "Healthy"
+        hs.message = "Waiting for analysis run to finish: status is successful."
+    end
+    if obj.status.status == "Failed" then
+        hs.status = "Degraded"
+        hs.message = "Waiting for analysis run to finish: status is failed."
+    end
+    if obj.status.status == "Error" then
+        hs.status = "Degraded"
+        hs.message = "Waiting for analysis run to finish: status is error."
+    end
+    if obj.status.status == "Inconclusive" then
+        hs.status = "Suspended"
```

Comment:
Should this be suspended? I'm thinking we don't return a health status because the analysisrun itself is not paused.